samples/ResourceOwnerPasswordFlow/Certificate.cer

```diff
@@ -0,0 +1,24 @@
+Bag Attributes
```

Comment:
FYI, .pfx certificates are now supported on CoreCLR. I removed the ugly workaround and `Certificate.cer`/`Certificate.keys` from the `Mvc.Server` sample.

I also replaced the unsafe workaround for Mono by a much safer approach that uses a RSA key instead of a X509 certificate. It has one downside though: the RSA key is held in memory and a new one is generated each time you start `Mvc.Server`, which causes validation exceptions on `Mvc.Client` if you restart the server without restarting the client.